packages/babel-helper-remap-async-to-generator/package.json

```diff
@@ -14,7 +14,9 @@
   },
   "main": "./lib/index.js",
   "dependencies": {
+    "@babel/core": "workspace:^",
```

Comment:
`@babel/core` should be moved to `peerDependencies`.

Reply:
I think there is no `peerDependencies` property in `packages/babel-helper-remap-async-to-generator/package.json`. Can I create one like `"peerDependencies": {"babel/core":"workspace:^"}` inside the file?